docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst

```diff
@@ -188,6 +188,10 @@ List of QoS Policy data members:
      :raw-html:`<br />`
      It cannot be changed on enabled entities.
 
+.. important::
```

Comment:
I'd add the counter part in reliability, stating that setting it to `BEST_EFFORT` makes the endpoints behave as `VOLATILE`